user_tools/src/spark_rapids_tools/cmdli/tools_cli.py

```diff
@@ -40,8 +40,8 @@ def qualification(self,
                       platform: str = None,
                       target_platform: str = None,
                       output_folder: str = None,
-                      filter_apps: str = None,
-                      estimation_model: str = None,
+                      filter_apps: str = QualFilterApp.tostring(QualFilterApp.get_default()),
+                      estimation_model: str = QualEstimationModel.tostring(QualEstimationModel.get_default()),
```

Comment:
Handling the default is done in the ArgProcessor class because there is some logic that is required to calculate the default.
That's why it is set to None in the CLI to be able to distinguish between whether the user left the argument empty or did he actually set the value intentionally.

Reply:
Thanks @amahussein! I will update this.